guillotina/routes.py

```diff
@@ -72,16 +80,25 @@ def __init__(self, raw_route):
             self.view_name = parts[0]
             self.route_parts.append(RoutePart(parts[0]))
             for part in parts[1:]:
-                self.view_name += "/"
-                if URL_MATCH_RE.match(part):
-                    self.route_parts.append(VariableRoutePart(part.strip("{").strip("}")))
+                if part.endswith(":path}"):
+                    self.view_name += "?"
+                    name = part.replace(":path", "").strip("{").strip("}")
+                    self.route_parts.append(PathRoutePart(name))
+                    break
```

Comment:
I don't think the break is necessary

Reply:
is to enforce that the "{xxx:path}" is always the last. Otherwise could be misleading if we allow the user to declare something like: "@api/{p:path}/hola" which is not valid, maybe we need to fail when we found this situation?

Reply:
oh right, duh!